src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Making the switch"
+tags:
+- Learning paths
+- Customer experience
+- Migration
+metaDescription: "How to fully migrate your data monitoring from Datadog to New Relic."
+---
+
+import journeyFullStack from 'images/journey_diagram_full-stack-monitoring.svg'
+
+If you've followed our tutorial series, you've learned a little about how New Relic works firsthand. You've instrumented your entities for infrastructure, applications, and logs to start really seeing how your stack works together. You've setup a premade dashboard to visualize the data across your stack, and you've set up an alert or two to help monitor issues that could arise with your systems. 
+
+<img
```

Comment:
To me the grid in this image looks pretty odd and distracting:
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/55203603/236565904-e449a7c2-1c59-4fcf-a562-1048fcb593d9.png">

Reply:
I'm not sure why it's showing up like that, it should have it's own background. I'll look into this, thank you!

Reply:
Fixed